apps/cowswap-frontend/src/modules/appData/updater/AppDataHooksUpdater.ts

```diff
@@ -46,18 +53,18 @@ export function AppDataHooksUpdater(): null {
       preInteractionHooks: permitData ? [permitData] : undefined,
     })
 
-    if (hooks) {
+    if (!isSmartContractWallet && !isNativeSell && hooks) {
```

Comment:
I guess this is already handled when you get the hook data (u just explained me how the generation return undefined in the case of native sell)

Reply:
Yes it should be.
But it's still getting through.